pkg/vault/vault.go

```diff
@@ -81,6 +81,10 @@ func New(c Config) Vault {
 
 func (v *vaultApi) getClient() (*api.Client, error) {
 	config := api.DefaultConfig()
+	if err := config.ReadEnvironment(); err != nil {
+		//If we cannot read environment variables fall-back to default config
+		fmt.Errorf("couldn't read config from environment: %v", err)
```

Comment:
This line as is here doesn't do anything. If you just want to print this text and continue, do this:
```
log.Printf("Couldn't read config from environment: %v\n", err)
```
If you want to return the error:
```
return nil, fmt.Errorf("couldn't read config from environment: %v", err)
```

We probably want to return the error, being silent in this kind of early startup config problems is evil :)